### PR TITLE
Create seats

### PR DESCRIPTION
**Pull request by Joannitka-pl**
- [x]  Created seats for hall.
- [x] Added seance functionality.
- [x] Added seat reservation logic.

### CODE REVIEW

File: app/concepts/seances/representers/list.rb [hunk 1]
```diff
@@ -1,17 +1,23 @@
 module Seances
   module Representers
     class List
-      attr_reader :seances
+      attr_reader :seances, :movie
```
Comment (Sonickers) [inline app/concepts/seances/representers/list.rb:4]: You added `:movie` but you don't use it 👀

File: db/seeds.rb [hunk 2]
```diff
@@ -13,4 +13,41 @@
 #                capacity: 100)
 # end
 
-# p 'Created #{Hall.count} halls.'
\ No newline at end of file
+# p 'Created #{Hall.count} halls.'
```
Comment (tobiaszwaszak) [inline db/seeds.rb:16]: try to not leave commented lines
if it redundant remove it
if it's important it is still in git history

File: app/concepts/seances/representers/list.rb (hunk 1 above)
Comment (Joannitka-pl) [reply app/concepts/seances/representers/list.rb:4]: That's right. Thank you 🙏

File: db/seeds.rb (hunk 2 above)
Comment (Joannitka-pl) [reply db/seeds.rb]: Will do. Thank you 😊

**Pull request merged**